torch/ao/quantization/quantizer/xnnpack_quantizer.py

```diff
@@ -236,6 +236,12 @@ def not_module_type_or_name_filter(n: Node) -> bool:
 
 
 class XNNPACKQuantizer(Quantizer):
+    """
```

Comment:
There are some decorator that allow you to declare either not bc compatible or something along the lines. @jerryzh168 do you know of any that can be used here?

Reply:
[this?](https://github.com/pytorch/pytorch/blob/adbbcd87d9fafe0aa34c69dd8f780f919a24461c/torch/fx/proxy.py#L40)

we can do that, looks like it modifies docstring: https://github.com/pytorch/pytorch/blob/adbbcd87d9fafe0aa34c69dd8f780f919a24461c/torch/fx/_compatibility.py#L12 might be useful if we mark it in a lot of things

Reply:
Ok doing (1) for now, *and* keeping this docstring.